app/assets/stylesheets/projects.css.scss

```diff
@@ -66,3 +66,13 @@
   font-style: italic;
   padding-left: 20px;
 }
+
+.nocounter {
+    color: darkgray;
+}
+
+.project.nocounter, p.nocounter {
```

Comment:
@hugopl, but if I remove these lines in one of the pages the behavior will be different, I mean... will not grayed out the link that doesn't return MR's. Like I did on project list page.  That's ok for you?